feature-libs/order/components/order-details/order-details-extended/consignment-tracking/consignment-tracking-link.component.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  inject,
+  OnDestroy,
+  OnInit,
+  ViewChild,
+  ViewContainerRef,
+} from '@angular/core';
+import {
+  LaunchDialogService,
+  LAUNCH_CALLER,
+  OutletContextData,
+} from '@spartacus/storefront';
+import { of, Subscription } from 'rxjs';
+import { take } from 'rxjs/operators';
+import { Consignment } from '@spartacus/order/root';
+import { OrderConsignmentsService } from '../../order-consignments.service';
+
+@Component({
+  selector: 'cx-consignment-tracking-link',
+  templateUrl: './consignment-tracking-link.component.html',
+  host: { class: 'cx-list-header col-12' },
```

Comment:
[QUESTION]
Did you consider putting `.col-12` explicitly on the host element, in the parent component's template? What pros/cons of alternative solutions do you see?

Why I ask: Now this component cannot be re-used outside the Bootstrap grid. So it would be good if we have other good arguments to justify the current solution.

Reply:
this is an implementation of outlet. so i suppose we cannot put it in parent component's template... how about using  @HostBinding('className') ?

Reply:
Ah, now understood. Because it's outlet component, there is no way to define css class in the parent component.

Could you leave a code comment near the `host: {....}  // ...` configuration, why we have to set those css classes here (becasue it's an outlet component).